### ISSUE TITLE
question: which source manipulation to use for stapling at the middle middle

### ISSUE THREAD

**Issue by animesh-sharama** [momijizukamori/bookbinder-js#58]
Hi I have the below content in the pdf for which I would like to generate output:

```
mypage_content_vertical = [
1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16
]
```
which `source manipulation` can be use to generate the output such that when printed on both side it can be stapled at the middle.

```
page 1: 
 front:
   first-half: page 2(start to center)
   second-half: page 15(center to end)
 rear:
   first-half: page 16(start to center)
   second-half: page 1(center to end)

page 2:
 front:
   first-half: page 4(start to center)
   second-half: page 13(center to end)
 rear:
   first-half: page 14(start to center)
   second-half: page 3(center to end)

page 3:
 front:
   first-half: page 6(start to center)
   second-half: page 11(center to end)
 rear:
   first-half: page 5(start to center)
   second-half: page 12(center to end)

page 4:
 front:
   first-half: page 8(start to center)
   second-half: page 9(center to end)
 rear:
   first-half: page 7(start to center)
   second-half: page 10(center to end)
```

**Comment by sithel** [comment]
I'll confess, I'm not 100% certain I'm reading your phrasing correctly for that page front/rear/1st half/2nd half parts...
BUT! seeing the 1 & 16 together, I think I get the idea --

If you have 16 pages and you want them to have 2 per side of paper, that's a FOLIO Page Layout

To make sure that they group them such that there's a single fold in the middle, that involves setting the SIGNATURE LENGTH down in the Signature Format section -- given the Folio layout, 1 sheet of paper has 4 pages (2 on each side). To have all 16 pages in one signature, you'll want 4 sheets (4x4 = 16). It defaults to 8 (which is suuuper big IMO) so you'll want to change that to 4. 

in summary: Folio Page Layout w/ Standard Signatures of length 4
(let me know if that works?)

**Comment by sithel** [comment]
🤦‍♀️ ok, you mentioned `source manipulation` -- I... don't think you need to do anything with that? 
am not sure what "start to end" means exactly -- but there are only a couple options, I recommend expanding them (click triangle) to see visualizations of what those layouts (once assembled) will look like
Also, you can use the Preview button to check the various rotations to see what they'd result in. Play around with it a bit- shouldn't take long to figure out what you want?

**Comment by Lottie16** [comment]
Hi @animesh-sharama ! I think, from how I understand your description, you’ll want the 90° anti-clockwise (counter clockwise) 

![IMG_0787](https://github.com/momijizukamori/bookbinder-js/assets/87568209/196621b8-b70d-4082-b587-dac19c79b320)

With the book/pamphlet open, this should give a straight read down the page from eg page 8 to page 9, with the spine across the centre - hope this helps?
